src/main/java/org/openrewrite/java/migrate/lang/var/package-info.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2021 the original author or authors.
```

Comment:
Looks like you're [not caught up with main](https://github.com/openrewrite/rewrite-migrate-java/commit/2f240932eae2acb366aab61fb89850412fbc23e2) yet.
```suggestion
 * Copyright 2023 the original author or authors.
```

Reply:
🫣 yes in very header..